src/react/listen.jsx

```diff
@@ -11,11 +13,19 @@ export default (listeners = defaultListeners) => WrappedComponent => class exten
   }
 
   componentWillMount() {
-    this.context.store.listeners.add(listeners)
+    const { store } = this.context
+
+    store.dispatch(toAction('ADDING'))
+    store.listeners.add(listeners)
+    store.dispatch(toAction('ADDED'))
```

Comment:
And is it really necessary to have `ADDING` and `ADDED` ?

Just having only `@@krm/LISTENERS>ADDED` maybe renamed in `@@krm/LISTENERS>MOUNTED`

Reply:
Listeners are not `mounted` this is the wrapped component that is mounted.

In first place I wrote:
**mount**
```js
store.listeners.add(listeners)
store.dispatch(toAction('ADDED'))
```

**unmount**
```js
store.dispatch(toAction('REMOVING'))
store.listeners.remove(listeners)
```

So you can catch both events into listeners that are linked.
But I didn't like that one is before, and the other after the linkage.

So I cloned them to have a "standard".